app/assets/javascripts/accordion-with-descriptions.js

```diff
@@ -210,14 +208,14 @@
         });
       }
 
-      function openStoredSections($section) {
+      function openStoredSections ($section) {
         toggleSection($section);
         toggleIcon($section);
         toggleState($section.parent().find('.subsection__button'));
         // setOpenCloseAllText();
       }
 
-      function setOpenCloseAllText() {
+      function setOpenCloseAllText () {
         var openSubsections = $('.subsection--is-open').length;
         // Find out if the number of is-opens == total number of sections
```

Comment:
I know this PR has not changed this, but if this is still an open question can we prefix with `TODO: ` so it is better recognised by IDEs?

Reply:
Okay - this linter has highlighted a lot of stuff that isn't mine...